flixel/FlxGame.hx

```diff
@@ -243,6 +243,14 @@ class FlxGame extends Sprite
 		// Super high priority init stuff
 		_inputContainer = new Sprite();
 		
+		if (GameSizeX <= 0)
```

Comment:
Sorry, I don't understand what do you mean by `windowSize x windowHeight`.
Another reason I can think of is to **reserve** the negative values for possible more implements in the future.